src/DiscordBot.js

```diff
@@ -148,37 +147,13 @@ class DiscordBot {
   }
 
   async updateBlacklist () {
-    if (!config.banServer) {
-      return false
+    if (!config.banServer) return
+    let blacklists = await global.Cache.get('blacklists', 'data')
+    if (!blacklists) {
+      blacklists = await request(`https://discord.com/api/v8/guilds/${config.banServer}/bans`, { json: true, headers: { Authorization: `Bot ${config.token}` } })
+      await global.Cache.set('blacklists', 'data', blacklists)
     }
-
-    const response = await request(`https://discord.com/api/v6/guilds/${config.banServer}/bans`, {
-      json: true,
-      headers: {
-        Authorization: `Bot ${config.token}`
-      }
-    })
-
-    response.forEach(ban => {
-      this.blacklist[ban.user.id] = true
-    })
-  }
-
-  async updateBlacklist () {
-    if (!config.banServer) {
-      return false
-    }
-
-    const response = await request(`https://discord.com/api/v6/guilds/${config.banServer}/bans`, {
-      json: true,
-      headers: {
-        Authorization: `Bot ${config.token}`
-      }
-    })
-
-    response.forEach(ban => {
-      this.blacklist[ban.user.id] = true
-    })
+    blacklists.forEach(ban => { this.blacklist[ban.user.id] = true })
```

Comment:
Can we fetch this in the main process instead, before launching any shards? Then we can fetch it from the blacklist here, and it will always be available. 

This code as it's written now may result in the bans being fetched multiple times. If multiple shards start at once, then a few requests will be made before it's put into the cache.